spec/helpers/example-data.ts

```diff
@@ -19,17 +19,17 @@ export const A_THIRD_NUMBER_STRING = A_THIRD_NUMBER.toString();
 
 export const A_DATE: Date = new Date('1997-07-04T16:56:55.000Z');
```

Comment:
Let's put a fraction on here so we are testing that behavior. So make it, like, 1997-07-04T16:56:55.454Z or something.

Reply:
Good catch!